app/config.local.js

```diff
@@ -29,3 +34,4 @@ kubernetes: "dev-mode"
 };
 
 window.MOCK_ORIGIN_SERVICES = false;
+window.MOCK_SERVICES_DATA = true;
```

Comment:
I was thinking we'd need to mock out certain sub-sets of data.  Perhaps, 'catalog services' isn't quite ready, so we'd mock it, whereas 'brokers' can be retrieved from DataServices.  So not an all-or-nothing, but 'a la cart'.